tern/command_lib/base.yml

```diff
@@ -379,3 +379,41 @@ npm:
           - "pkgs=`npm list -g | sed 's/\\(^[^a-z]*\\)\\([_.a-z0-9-]*\\)\\(@.*\\)/\\2/g' | grep -v '/usr*' | uniq`"
           - "for p in $pkgs; do npm view $p repository.url; done"
     delimiter: "\n"
+# golang----------------------------------------------------------------------
+go:
+  pkg_format: ''
+  os_guess:
+    - 'None'
+  path:
+    - 'usr/local/go/bin'
+  pkg_separators:
+    - '@'
+    - '@^'
+    - '@~'
+  pinning_separator: ':'
+  names:
+    invoke:
+      1:  
+        container:
+          - "/usr/local/go/bin/go list -m all | tail -n +2 | cut -d ' ' -f 1" 
+    delimiter: "\n"
+  versions:
+    invoke:
+      1:
+        container:
+          - "/usr/local/go/bin/go list -m all | tail -n +2 | cut -d ' ' -f 2"
+    delimiter: "\n"
+  copyrights:
+    invoke:
+      1:
+        container:
+          - "export GOPATH=/go"
```

Comment:
While the default `$GOPATH` environment variable for the docker-library golang image is `/go`, it can be changed. I propose setting this environment variable based on the value of the `GOPATH` value in the config and if it's not available there, using `/go` as the default. I will open a PR for this shortly.

Reply:
Opened https://github.com/tern-tools/tern/pull/780